### PR TITLE
Update sunbird_actor_servc.md

### PR DESCRIPTION
**Pull request by manzarul**
Different modes for Actor service configuration.

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Hello @manzarul,<br/> _This is a auto-generated response._ <br/> Click the URL for review pull request - http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/178.

**Comment by lakhanmandloi** [comment]
Hello @manzarul,<br/> _This is a auto-generated response._ <br/> Click the URL for review pull request - http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/178.

**Comment by lakhanmandloi** [comment]
Hello @manzarul,<br/> _This is a auto-generated response._ <br/> Click the URL for review pull request - http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/178.

**Comment by lakhanmandloi** [comment]
Hello @manzarul,<br/> _This is a auto-generated response._ <br/> Click the URL for review pull request - http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/178.

File: pages/developer-docs/installation/sunbird_actor_servc.md [hunk 1]
```diff
@@ -1,48 +1,48 @@
 
-## Requirement
+## Prerequisites
 
-System should have the ability to run the akka actor in local ,remote or mixed mode. Based on the configuration values system can run akka actor either in local or remote mode in similar way background job actor can also be run any one of the mode. 
+- Ensure that the system has the ability to run the akka actor in local ,remote or mixed mode
```
Comment (nawabahmadreshi) [inline pages/developer-docs/installation/sunbird_actor_servc.md]: @Basreena can you please re-review this for language and Grammar.

**Comment by nawabahmadreshi** [comment]
Hi @manzarul, review the following so that I push it to production
http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/review/developer-docs/installation/sunbird_actor_servc/

**Pull request merged**